Update Contribution Guide (#3269)
windows-toolkit/WindowsCommunityToolkit: contributing.md

Once again Welcome aboard and have fun in this adventurous journey of Windows Community Toolkit :raised_hands:

In the next few steps you will be able to learn how to contribute to this amazing community, but before you proceed any further it is good to understand what Widows Community Toolkit is all about. 

The foundation of the **Windows Community Toolkit** is simplicity. 

A developer should be able to quickly and easily learn to use the API.

Simplicity and a low barrier to entry are must-have features of every API. If you have any second thoughts about the complexity of the design, it is almost always much better to cut the feature from the current release and spend more time to get the design right for the next release.

You can always add to an API, you cannot ever remove anything from one. If the design does not feel right, and you ship it anyway, you are likely to regret having done so.

That's why many of the guidelines of this document are obvious and serve only one purpose: Simplicity.

> Here is how you can contribute to the Windows Community Toolkit. 

Reviewer: We should probably start a new section of 'How to Contribute to the Community Toolkit' and list out the high-level things like reviewing docs, testing a PR, etc... and make some of the list below hidden behind sub-bullets or maybe we have a separate page all around actually contributing a feature to the Toolkit (as that's more involved)?

Author: WCT Wiki has solved this issue.

WCT has a list of issues that are labeled as [help wanted](https://github.com/windows-toolkit/WindowsCommunityToolkit/labels/help%20wanted%20%3Araising_hand%3A). The level of complexity in the list can vary but if you have an advanced level of programming experience, feel free to jump in to solve these issues.

## <a name="feature"></a>Add New Feature :mailbox_with_mail:
*	To contribute a new feature, fill out the [Feature Request Template](https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/new?assignees=&labels=feature+request+%3Amailbox_with_mail%3A&template=feature_request.md&title=%5BFeature%5D) and provide detailed information to express the proposal. 
*	Once the Feature Request is submitted, it will be open for discussion.
*	If it gets approved by the team, proceed to submit a PR of the proposed Feature. 
*	If the PR contains an error-free code and the reviewer signs off, the PR will be merged.

## <a name="docs"></a> Add or Improve Documentation :page_with_curl:

Due to the involvement of multiple steps to add or improve documents; it is required to visit [Windows Community Toolkit Wiki](https://aka.ms/wct/wiki) and follow contribution guidelines.

>  ## <a name="pr"></a>Create, Submit or Review Pull Request :rocket:

Reviewer: ```suggestion
## <a name="pr"></a>Create, Submit or Review Pull Request :rocket:
```

Extra space

# Contributing to the Windows Community Toolkit :sparkles::sparkles:

> The Windows Community Toolkit team is delighted to welcome you for exhibiting interest in contributing to one of the most exhilarating and growing projects. Any contribution or value added will go a long way to enhance the project.

Reviewer: ```suggestion
Thank you for exhibiting interest in contributing to the Windows Community Toolkit. The team is delighted to welcome you onboard to our exciting and growing project. Any contribution or value added go a long way to enhance the project!
```

In the next few steps you will be able to see a glimpse of ways you can contribute to the Windows Community Toolkit

> :rotating_light: **It is highly recommended to visit [Windows Community Toolkit Wiki](https://github.com/windows-toolkit/WindowsCommunityToolkit/wiki) where you can find complete and detail-oriented content of this page** :rotating_light:

Reviewer: ```suggestion
:rotating_light: **It is highly recommended to visit [Windows Community Toolkit Wiki](https://aka.ms/wct/wiki) where you can find complete and detail-oriented content of this page** :rotating_light:
```

> ## <a name="question"></a> Questions :grey_question:

Reviewer: Let's keep the table of contents as a starting point. e.g.

## Ways to Contribute

There are many ways to contribute to the project besides submitting or fixing issues. Below are a list of links to topics in our Wiki on each subject of how you can get started.

... (TOC from before Questions, Docs, etc...)

A bit more like https://github.com/microsoft/vscode/blob/master/CONTRIBUTING.md

Author: I think with the current structure, it looks good now?

## <a name="files"></a> Files and folders
* DO associate no more than one class per file.
* DO use folders to group classes based on features.

> :rotating_light: **Please visit [Windows Community Toolkit Wiki](https://github.com/windows-toolkit/WindowsCommunityToolkit/wiki) to view complete and detail-oriented content of this page** :rotating_light:

Reviewer: Liked the Thank You at the bottom the VS Code had.
